docs/core/testing/mstest-analyzers/mstest0007.md

```diff
@@ -32,10 +32,11 @@ A method that's not marked with <xref:Microsoft.VisualStudio.TestTools.UnitTesti
 
 The following test attributes should only be applied on methods marked with the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute> attribute:
 
+- <xref:Microsoft.VisualStudio.TestTools.UnitTesting.ConditionBaseAttribute>
```

Comment:
This is not going to work until https://github.com/MicrosoftDocs/visualstudio-docs-dotnet-api-pr/pull/212 is published